widgets/contact/contact.php

```diff
@@ -712,19 +721,24 @@ function modify_instance( $instance ) {
 	}
 
 	function get_template_variables( $instance, $args ) {
-		$vars = array();
-
 		unset( $instance['title'] );
 		unset( $instance['display_title'] );
 		unset( $instance['design'] );
 		unset( $instance['panels_info'] );
 
 		// Include '_sow_form_id' in generation of 'instance_hash' to allow multiple instances of the same form on a page.
-		$vars['instance_hash'] = md5( serialize( $instance ) );
-
+		$instance_hash = md5( serialize( $instance ) );
 		unset( $instance['_sow_form_id'] );
 
-		return $vars;
+		if ( ! empty( $instance['settings']['submit_attributes']['submit_id'] ) ) {
```

Comment:
I don't see `submit_attributes` or `submit_id` being defined anywhere? Is there a commit you forgot to push?

Reply:
Last second change. 😞